starters/gatsby-starter-minimal/gatsby-config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+    siteMetadata: {
+        siteUrl: `https://www.yourdomain.tld`,
```

Comment:
`create-gatsby` doesn't have the ability (yet) to define siteMetadata data for each plugin so for now this must be in here for everything (even when doesn't use sitemap plugin)

Reply:
I've got siteMetadata defined in config but still got this issue.